### ISSUE TITLE
Synthetic indexes value in different devices

### ISSUE THREAD

**Issue by martinrf79** [binary-com/binary-bot#2424]
Hello how are you, a query for all about synthetic indexes. Let's say there are 4 accounts opened in synthetic indexes 10 for example, in different geographical locations and in different devices. My question is whether the 4 accounts will have exactly the same value, or varies according to each account. I am seeing some aspects that are beginning to generate doubts about the broker.

**Comment by ZDEGLET** [comment]
I also thought about it. easier to negotiate with 4 interested and check. I agree to do a check.
because you need to turn on robots at the same time and then check the statistics.
if you write something (the subject of the letter STATISTIKA) we will agree on the mail.
zdeglet@gmail.com

**Comment by hotmatrixx** [comment]
> 
> 
> Hello how are you, a query for all about synthetic indexes. Let's say there are 4 accounts opened in synthetic indexes 10 for example, in different geographical locations and in different devices. My question is whether the 4 accounts will have exactly the same value, or varies according to each account. I am seeing some aspects that are beginning to generate doubts about the broker.

answer;
synthetic indexes are synthetic. made up. fictional. they are numbers generated by the broker, and have nothing to do with 'real' markets.

If you want to play with them, go ahead, But if you'd like more consistency and reliable/ accountable numbers that tend to sync up with the real markets, then yeah... use real markets.

**Comment by martinrf79** [comment]
Yes, I agree with what you say, although if the algorithm or ia that generates the prices that the broker has is not the "same" for all users, be it manual or bot, real account or demo, then that's where Doubts appear.

**Comment by trident2005** [comment]
I have set up two different accounts in two different internet providers simultaneously and the values are exactly the same, they could not be different by users that would create a conflict with the copytrade, it is my opinion

**Comment by thommmaz** [comment]
> 
> 
> Hello how are you, a query for all about synthetic indexes. Let's say there are 4 accounts opened in synthetic indexes 10 for example, in different geographical locations and in different devices. My question is whether the 4 accounts will have exactly the same value, or varies according to each account. I am seeing some aspects that are beginning to generate doubts about the broker.

Lately I was started thread about price differences, price can be different even in 1 device, don't expect to get same price in 4 devices... broker is always 1 step ahead of you.
![70697239-d70c4380-1ccd-11ea-937d-3a98888a4ba6](https://user-images.githubusercontent.com/40931373/74037006-13a2a580-49c6-11ea-9685-9897d5327421.png)

https://github.com/binary-com/binary-bot/issues/2233#issue-536840261

**Comment by trident2005** [comment]
> 
> 
> > Hello how are you, a query for all about synthetic indexes. Let's say there are 4 accounts opened in synthetic indexes 10 for example, in different geographical locations and in different devices. My question is whether the 4 accounts will have exactly the same value, or varies according to each account. I am seeing some aspects that are beginning to generate doubts about the broker.
> 
> Lately I was started thread about price differences, price can be different even in 1 device, don't expect to get same price in 4 devices... broker is always 1 step ahead of you.
> ![70697239-d70c4380-1ccd-11ea-937d-3a98888a4ba6](https://user-images.githubusercontent.com/40931373/74037006-13a2a580-49c6-11ea-9685-9897d5327421.png)
> 
> [#2233 (comment)](https://github.com/binary-com/binary-bot/issues/2233#issue-536840261)

Try with one tick, the close spot could change  after several ticks

**Comment by muroyal** [comment]
Hello Friends,

Please try to test with 1 minute candles trade at least we can try with 1 minute trades  ::::::::::: ?

**Comment by muroyal** [comment]
virtual account same bot same time it will execute.. here is the results. :(

![image](https://user-images.githubusercontent.com/32416314/74103839-6ff7f780-4b75-11ea-98a6-68ac15526f7e.png)


![image](https://user-images.githubusercontent.com/32416314/74103856-9158e380-4b75-11ea-80e7-14429beeb57a.png)

**Comment by aaron-binary** [comment]
The feed isn't different based on account type or geographical location, it'll be exactly the same for everyone. The differences above will most likely have to do with the exit times being different from each other.

**Comment by muroyal** [comment]
@aaron-binary Exit time is same given 55 seconds.. on both bots. I have ran the bots entire night the virtual account trades are more I mean executed 30 trades and the real accounts trades are executed 32 time and lost 10$ in real account and virtual account survived and got 7 $ profit. 

I have ran the bots on same conditions and with 55 seconds trade on every new candle. That means the bots should start on same time and exit at same time ..
